tools/slack.md

```diff
@@ -40,41 +40,41 @@ You can find other users in the Slack organization on the left-menu below the ch
 
 You can find a list of Channels on the left menu. Each channel is a group conversation with one or more users. Each channel typically exists for a specific topic.
 
-At ada we use a few channels for specific things:
-
-- _annoucements_ - Used for Ada official annoucements about events, or curriculum.
-- _csmemes_ - a channel to share funny memes or jokes about programming or technology.
-- _pre-ada-homework_ - Used for discussions on the Pre Ada Material
+You were automatically added to a number of channels when you joined the Slack for your cohort, such as `#announcements`, `#general`, and `#study-hall`. Check the description of each channel to see what it's for.
 
 ### Creating Channels
 
 You are welcome to create channels for discussions on a topic of interest. With discussions in Slack please remember and adhere to the Code of Conduct Expectations.
 
-Like with users you can use the `+` sign to join channels you aren't currently in.
+After adding a channel, it's usually a good idea to make a post in `#general` to let other folks know the channel exists. Otherwise, it's likely that no one will know about it!
 
-## Customizing Slack
+As with users you can use the `+` sign to join channels you aren't currently in. Before making your own channel, check to see if there is already a channel for the topic you are interested in.
 
-You are welcome to customize slack by adding emoji.
+## Customizing Slack
 
-![Customize Slack](../assets/customize-slack.png)
+There are many settings in Slack that can be customized to your liking. Many settings can be accessed by clicking the `Settings...` option under the `Slack` menu item listed in the menu at the top of the desktop.
 
-Feel free to experiment and try settings in Slack.
+There are also additional settings that can be changed by other means.
 
 ### Adding Emojis
 
-You can also add your own custom emoji to Slack. You are encouraged to do so, this Slack organization belongs to your cohort, and you are encouraged to make it your own.
+You can add your own custom emoji to Slack. You are encouraged to do so, this Slack organization belongs to your cohort, and you are encouraged to make it your own. Use the `Add Emoji` button displayed in the emoji picker to upload your own emoji.
+
+![Add Emoji button in the emoji picker](../assets/add-emoji.png)  
+*Fig. The emoji picker, where we can access the Add Emoji button*
 
-![Customize Slack Emojis](../assets/customize-slack2.png)
+![User experience for uploading additional emoji](../assets/add-emoji2.png)  
```

Comment:
Would it be possible to retake the screenshots in dark mode? I'd rather get them switched over now, but if time's too short we can make a ticket to revisit the images.